docs/en/observability/create-observability-connectors.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[cases-external-connectors]]
+= Configure external connectors
+
+You can send cases to these third-party systems:
+
+* ServiceNow ITSM
+* ServiceNow SecOps
+* Jira (including Jira Service Desk)
+* IBM Resilient
+* {swimlane}
+
+IMPORTANT: To send cases to external systems, you need the appropriate license, and your role must
+have the *Cases* {kib} privileges as a user. For more details, refer to <<grant-cases-access>>.
+
+You need to create a connector to send cases, which stores the information required to interact
+with an external system.
+
+After creating a connector, you can set your cases to
+automatically close when they are sent to an external system.
+
+[discrete]
+[[new-connector-observability]]
+== Create a connector
+
+. Go to *Cases* -> *Edit external connection*.
++
+[role="screenshot"]
+image::images/add-case-connector.png[]
+. From the *Incident management system* list, select *Add new connector*.
+. Select the system to send cases to: *ServiceNow*, *Jira*, *IBM Resilient*, or *{swimlane}*.
+
+. Enter your required settings.
++
+|=== 
+
+| *Connector name* | Name for the connector. 
+
+| *URL* | The URL of the external system to which you want to send cases.
+
+| *API URL* | The URL of the Swimlane instance to which you want to send cases ({swimlane} only).
+
+| *Organization ID* | Your organization’s IBM Resilient ID number (IBM Resilient only).
+
+| *Application ID* | The application ID of your {swimlane} application. From {swimlane}, you can find the application
+ID by checking your application’s settings or at the end of your application’s URL after you’ve opened it ({swimlane} only).
+
+| *Username* | The username of the ServiceNow account is used to access the ServiceNow instance (ServiceNow only).
+
+| *Password* | The password of the ServiceNow account is used to access the ServiceNow instance (ServiceNow only).
+
+| *Project key* | The key of the Jira project to which you are sending cases (Jira only).
+
+| *Email or Username* | The Jira account username or email (Jira only).
+
+| *API token or Password* | The API token or password is used to authenticate Jira updates (Jira only).
+
+| *API key ID* | The API key is used to authenticate IBM Resilient updates (IBM Resilient only).
+
+| *API key secret* | The API key secret is used to authenticate IBM Resilient updates (IBM Resilient only).
+
+| *API token* | The {swimlane} API authentication token is used for HTTP Basic authentication ({swimlane} only).
```

Comment:
Is it worth putting the e.g. (ServiceNow only) callouts at the start of the sentences rather than end?

Reply:
Great suggestion. Thanks.